lib/active_admin/resource/attributes.rb

```diff
@@ -0,0 +1,44 @@
+module ActiveAdmin
+
+  class Resource
+    module Attributes
+
+      def default_attributes
+        resource_class.columns.each_with_object({}) do |c, attrs|
+          unless reject_col?(c)
```

Comment:
* Regarding `_count` columns. Ok, we can consider that the behavior you're adding are bug fixes since some users are complained about it, but this new behavior would be different since nobody has asked for it. So agreed, let's welcome this little extra complexity for now for the sake of keeping the current behavior.

* Regarding the `id_column` method, you're absolutely right, I completely overlooked that. I just applied the little patch and no feature specs broke so I incorrectly assumed it wasn't necessary.

So, agreed, let's not make any of the changes I suggested in https://github.com/activeadmin/activeadmin/pull/4784#discussion_r105568747. I :-1:'d my comment as well! :rofl: